src/components/QuestionPage.js

```diff
@@ -206,67 +238,161 @@ class QuestionPage extends Component {
       );
     }
   }
-  renderItem = ({item}) => {
-    return (
-      <TouchableOpacity
-        disabled={this.disable()}
-        onPress={() => this.isClicked(item)}>
-        <View style={this.answerStyle(item)}>
-          <View>
-            <Text
-              style={{
-                textAlign: 'center',
-                fontSize: 15,
-                fontWeight: 'bold',
-                marginHorizontal: 10,
-                flexShrink: 1,
-              }}>
-              {item}
-            </Text>
-          </View>
-        </View>
-      </TouchableOpacity>
-    );
-  };
+
+  useJoker() {
+    const {questionReducer} = this.props;
+
+    this.setState({
+      isJoker: true,
+    });
+
+    var answerslist = questionReducer.activeQuestion[ANSWER_INDEX];
+
+    var i;
+    var newanswerslist = [];
```

Comment:
Var kullanmak daha doğru let kullanalım.

Reply:
Dışarıda oluşturduğum için, "let" kullandığım takdirde farklı kod bloğunda; örneğin 249. satırdaki for döngüsünde kullanamazmışım gibi geldi 😐